subworkflows/local/mitochondria/convert_mt_bam_to_fastq.nf

```diff
@@ -2,23 +2,26 @@
 // Prepare bam files for MT allignment
 //
 
-include { SAMTOOLS_VIEW as SAMTOOLS_VIEW_MT       } from '../../../modules/nf-core/samtools/view/main'
+include { GATK4_PRINTREADS as GATK4_PRINTREADS_MT } from '../../../modules/local/gatk4/printreads/main'
```

Comment:
What is the advantage of using GATK4_PRINTREADS instead of SAMTOOLS_VIEW?

Reply:
Printreads is used in gatk's mitochondrial workflow, and its gives us the option to use readable filters. Otherwise functionally it makes no difference as they can both extract reads from a bam file.

Reply:
I much prefer samtools to gatk. But that might just be my dislike for java shining through